package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tbd54566975/dwn-sdk-js",
-  "version": "0.0.17",
+  "version": "0.0.18",
```

Comment:
is the `main` entrypoint still on the table to add?

Reply:
just added! need to test and make sure it works now

Reply:
I can confirm the `main` entrypoint works however wonder if you definitely want cjs given that react native should work with esm and via the `react-native` directive instead of `main`